app/public/cantusdata/templates/admin/map_folios.html

```diff
@@ -343,14 +350,21 @@ <h6><em>Drag and drop a CSV file anywhere<br>on the page to import it.</em></h6>
             var lines = e.target.result.split('\n');
             var inputIndex = 0;
             for (var i = 1; i < lines.length; i++) {
-                var values = lines[i].split(',');
-
+                let line = lines[i].replaceAll('\r','');
+                let values = line.split(',');
+                
+                // Iterate through visible inputs until the appropriate image uri
+                // is reached. In manuscripts with images with two folios (and
+                // therefore two input fields per image), the fields are filled in the
+                // order in which they occur in the backup file.
                 while (inputIndex < visibleInputs.length && visibleInputs[inputIndex].name !== values[1])
```

Comment:
if `values[1]` and `[0]` were unpacked just below `let values` with variable names indicating their meaning, this could read more clearly!